manifests/operatorconfiguration.crd.yaml

```diff
@@ -317,6 +317,8 @@ spec:
                       type: string
                   watched_namespace:
                     type: string
+              read_only_root_filesystem:
+                type: boolean
```

Comment:
Please move under `kubernetes` section and follow alphabetical sorting. Can you also add it [here](https://github.com/zalando/postgres-operator/blob/master/charts/postgres-operator/crds/operatorconfigurations.yaml) and [here](https://github.com/zalando/postgres-operator/blob/695ad44caffa7c03e19a1b59796d93942d8cb70c/pkg/apis/acid.zalan.do/v1/crds.go#L1175).